playground/frontend/pubspec.yaml

```diff
@@ -45,7 +45,10 @@ environment:
 # versions available, run `flutter pub outdated`.
 dependencies:
   aligned_dialog: ^0.0.6
-  code_text_field: ^1.0.0
+  code_text_field:
+    git:
+      url: https://github.com/BertrandBev/code_field.git
+      ref: 9e2c9fe52a69481f038f4b6609e8a0a776429437
```

Comment:
This PR relies on the recent change we made to the code editor package we are using. They are merged to the package's `master` but are not yet released as a new version on pub.dev. The maintainer is slow in communication. Is it OK to refer to the package like this for now? We tested this commit.

Reply:
The maintainer has promised to publish it on pub.dev in a couple of days.